core/tests/Test.Microsoft.Identity.LabInfrastructure/LabUserHelper.cs

```diff
@@ -56,7 +56,7 @@ public static UserQueryParameters DefaultUserQuery
             }
         }
 
-        public static ILabResponse GetUser(UserQueryParameters query)
+        public static LabResponse GetLabUserAndApp(UserQueryParameters query)
```

Comment:
How about GetLabUserData? right now, we are only using the appid from the Lab response but there are many more properties that we may end up using. (like tenantId)

Reply:
Sure, will do. The main refactoring was removing some of the interfaces, because they were not really doing much but were complicating things when it came to fetch the password etc.